src/CommandLine/Endpoint.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus.CommandLine
+{
+    using System;
+    using System.Threading.Tasks;
+    using McMaster.Extensions.CommandLineUtils;
+    using Microsoft.Azure.ServiceBus.Management;
+
+    static class Endpoint
+    {
+        public static async Task Create(ManagementClient client, CommandArgument name, CommandOption topicName, CommandOption subscriptionName, CommandOption<int> size, CommandOption partitioning)
+        {
+            try
+            {
+                await Queue.Create(client, name, size, partitioning);
+            }
+            catch (MessagingEntityAlreadyExistsException)
+            {
+                Console.WriteLine("Queue already exists, skipping creation");
```

Comment:
I think it would have been helpful to include the name of the queue that couldn't be created

Reply:
We probably don't want to bother until this is requested by those that use the tool.
From what we've found, the tool was not very much in [demand](https://www.nuget.org/packages/NServiceBus.Transport.AzureServiceBus.CommandLine/).

Reply:
Seems like it would be worth adding.

Reply:
I'm questioning the value, but would suggest to raise a separate issue as a maintenance release not to bog down the release.

Reply:
Replaced with issue #30